daffodil-runtime1/src/main/scala/org/apache/daffodil/processors/VariableMap1.scala

```diff
@@ -148,7 +154,11 @@ class VariableInstance private (
     this._value = Some(v)
   }
 
-  def setDefaultValue(v: DataValuePrimitiveNullable) = this._value = Some(v)
+  /* This is used to set a default value without assigning a priorValue */
```

Comment:
I noticed forceExpressionEvaluations uses setValue. Should it be using setDefaultValue? Or is this something that should realy only be used at runtime?

Reply:
You're right, forceExpressionEvaluations should be using setDefaultValue.